### PR TITLE
Add latency to the traffic stats

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add latency to the traffic stats
- Rename: EventCounter -> TrafficDatabase
- Update grafana.json

### CODE REVIEW

File: traffic/traffic-stats-manager.js [hunk 1]
```diff
@@ -1,52 +1,57 @@
...
-  addEvent(eventType, currentTimeMs = null) {
+  addEvent(eventType, latencyMs, currentTimeMs = null) {
     if (!this.enabled) {
       return;
     }
-    if (!this.eventCounterMap.has(eventType)) {
-      const newTdb = new EventCounter(this.intervalMs, this.maxIntervals, currentTimeMs);
-      this.eventCounterMap.set(eventType, newTdb);
+    if (!this.trafficDbMap.has(eventType)) {
+      const newTdb = new TrafficDatabase(this.intervalMs, this.maxIntervals, currentTimeMs);
```
Comment (liayoo) [inline traffic/traffic-stats-manager.js:16]: minor) `newTDb` or `newTrafficDb` ?

**Review by liayoo — APPROVED** [review]
LGTM with a minor comment. Thanks!

File: traffic/traffic-stats-manager.js (hunk 1 above)
Comment (platfowner) [reply traffic/traffic-stats-manager.js]: Done, thanks!

**Comment by platfowner** [comment]
Thanks for the fast review! Now merging..

**Pull request merged**